python/cudf/cudf/core/frame.py

```diff
@@ -3357,6 +3394,12 @@ def _scan(self, op, axis=None, skipna=True):
         2   6  24
         3  10  34
         """
+        if isinstance(self, cudf.BaseIndex):
+            warnings.warn(
+                "This method is deprecated and will be removed.",
```

Comment:
Does `op` map to the correct method name?
```suggestion
                f"Index.{op} is deprecated and will be removed.",
```

Reply:
Ooh sneaky move. Yes that will work.